tests/snmp/test_snmp_fdb.py

```diff
@@ -92,6 +92,39 @@ def is_port_channel_up(duthost, config_portchannels):
     return True
 
 
+def check_snmp_facts(duthost, localhost, hostip, creds_all_duts, config_portchannels, send_cnt, send_portchannels_cnt):
+    dummy_mac_cnt = 0
+    recv_portchannels_cnt = 0
+    snmp_facts = get_snmp_facts(
+        duthost, localhost, host=hostip, version="v2c",
+        community=creds_all_duts[duthost.hostname]["snmp_rocommunity"], wait=True)['ansible_facts']
+    if 'snmp_fdb' not in snmp_facts:
+        return False
```

Comment:
Minor comment, in the previous test case, the assert log message was descriptive enough to understand the failure reason, can we add an info log here and other places before returning False.